### PR TITLE
Adding temporary Orientation Page

### PR DESCRIPTION
**Pull request by rebeccawilliams**
[Preview link](https://cg-f5a56614-c88e-470c-9f06-ff5fe2d4c98a.app.cloud.gov/preview/gsa/resources.data.gov/orientation/)

### CODE REVIEW

**Comment by adborden** [comment]
FYI the preview link is https://cg-f5a56614-c88e-470c-9f06-ff5fe2d4c98a.app.cloud.gov/preview/gsa/resources.data.gov/orientation/

I'm going to tweak our contributing docs.

File: pages/orientation.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Evidence Act Orientation
+layout: page
+primary_nav_section: Evidence Act Orientation
+---
+
+# Evidence Act Orientation
```
Comment (adborden) [inline pages/orientation.md]: The page layout already includes the title as an H1, so we'll want to delete this line to avoid repeating the heading.

**Comment by adborden** [comment]
To find the preview URL, at the bottom of the PR, click "show all checks" and then "details" to the right of the Federalist build check.

File: pages/orientation.md (hunk 1 above)
Comment (rebeccawilliams) [reply pages/orientation.md]: Done!

**Comment by rebeccawilliams** [comment]
The previews look great! @adborden can you merge this when you have a chance so I am not merging my own PR?

**Pull request merged**